test/app-luatest/trigger_module_test.lua

```diff
@@ -1,7 +1,20 @@
+local server = require('luatest.server')
 local t = require('luatest')
 
 local g = t.group()
 
+g.after_each(function()
+    local trigger = require('trigger')
```

Comment:
You are using tarantool which is used as a **test runner** (this may affect other tests during a competitive launch).

Maybe we'll switch to a separate server instance?

Reply:
Thank you, moved the tests to a separate server in the first commit.